code/client/launcher/CitizenGame.cpp

```diff
@@ -437,7 +437,13 @@ void CitizenGame::Launch(const std::wstring& gamePath, bool isMainGame)
 	}
 
 	// force loading _this_ variant
-	LoadLibrary(MakeRelativeCitPath(L"scripthookv.dll").c_str());
+	HMODULE scripthookvHandle = LoadLibrary(MakeRelativeCitPath(L"scripthookv.dll").c_str());
+
+	if (!scripthookvHandle)
+	{
+		FatalError("Could not load scripthookv.dll.");

```

Comment:
Error states ought to be ignored here - this is just to ensure dependent .asi files will load our variant, if present.

The comment could be made more clear instead, since the plugin not being present will fail later during ComponentLoader (and this change will break launching RDR).